src/app/components/search/SearchField.js

```diff
@@ -8,23 +8,22 @@ import Grid from '@material-ui/core/Grid';
 import IconButton from '@material-ui/core/IconButton';
 import Button from '@material-ui/core/Button';
 import Popover from '@material-ui/core/Popover';
-import CloseIcon from '@material-ui/icons/Close';
 import Dialog from '@material-ui/core/Dialog';
-import SearchIcon from '@material-ui/icons/Search';
-import { Clear as ClearIcon } from '@material-ui/icons';
 import MediaPreview from '../feed/MediaPreview';
-import { borderWidthLarge } from '../../styles/js/shared';
+import SearchIcon from '../../icons/search.svg';
+import ClearIcon from '../../icons/clear.svg';
+import OpenInFullIcon from '../../icons/open_in_full.svg';
 
 const useStyles = makeStyles(theme => ({
   input: {
     borderRadius: 50,
     fontSize: 14,
   },
   inputInactive: {
-    border: `${borderWidthLarge} solid var(--grayBorderMain)`,
+    border: '2px solid var(--grayBorderMain)',
```

Comment:
reviewers note. Eventually we will remove the vars in shared.js and update the search component with its own css file, so I made this change and at tiny step in that direction, but kinda unrealated to the icons slipped in while i was working